ArduPlane/tailsitter.cpp

```diff
@@ -597,7 +597,7 @@ bool Tailsitter::in_vtol_transition(uint32_t now) const
     if (transition->transition_state == Tailsitter_Transition::TRANSITION_ANGLE_WAIT_VTOL) {
         return true;
     }
-    if ((now != 0) && ((now - transition->last_vtol_mode_ms) > 1000)) {
+    if ((now != 0) && ((now - transition->last_vtol_mode_ms) > 2000)) {
```

Comment:
This 1 second is really just to fix a race condition between mode change and transition update changing the transition state. Why does it need to be changed here?